api/src/main/java/org/jboss/forge/roaster/Roaster.java

```diff
@@ -93,36 +99,58 @@ private Roaster()
             return result;
          }
       }
-      throw new ParserException("Cannot find JavaParserProvider capable of producing JavaSource of type "
-               + type.getSimpleName(), new IllegalArgumentException(type.getName()));
+      throw new ParserException("Cannot find JavaParser capable of parsing the requested data"); 
```

Comment:
Why change the exception message here?

Reply:
Good question. Happened while coping code :-)